packages/orchestration/src/utils/mockChainInfo.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file Mocked Chain Info object until #8879
```

Comment:
UNTIL is a handy notation, but I'm uneasy that there's little to stop us from closing issues without addressing them.

maybe a back-link to this PR will help a little:

 - #8879